build.gradle

```diff
@@ -20,6 +20,7 @@ repositories {
 
 dependencies {
     compileOnly 'com.gradle:gradle-enterprise-gradle-plugin:3.13'
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
Can we use Junit 5?

Reply:
I've never used it and dunno how

Reply:
[9580449](https://github.com/gradle/common-custom-user-data-gradle-plugin/pull/169/commits/95804491c3badf954318461205fe63e77a96c119)